jobrunner/models.py

```diff
@@ -325,9 +325,7 @@ def started_at_isoformat(self):
     def completed_at_isoformat(self):
         return timestamp_to_isoformat(self.completed_at)
 
-    # On Python 3.8 we could use `functools.cached_property` here and avoid
-    # recomputing this every time
-    @property
+    @cached_property
     def slug(self):
```

Comment:
I think we could actually get rid of `slug` altogether. I don't think we use it anywhere now.

Reply:
hilariously, its used in kill_job and retry_job.

But yeah, may do that as a follow on